log.c

```diff
@@ -63,7 +63,7 @@ void get_timestr(char *timestring)
 
 void __attribute__ ((format (gnu_printf, 1, 2))) logg(const char *format, ...)
 {
-	char timestring[32] = "";
+	char timestring[84] = "";
```

Comment:
This change ensures that the string will be large enough to hold anything that might be produced by
https://github.com/pi-hole/FTL/pull/536/files#diff-f8cdfce5cea83a80a9847ba558991869R61